.k8s/production/testnet/dukong/kustomization.yaml

```diff
@@ -81,6 +82,21 @@ helmCharts:
           protocol: TCP
           name: http
     deployment:
+      initContainers:
+        - name: migrate-db
+          image: alpine/psql:16.3
+          command:
+            - /usr/bin/psql
+            - postgres://resolute:${userPassword}@postgresql:5432/resolute
+            - -f
+            - /tmp/update_schema.sql
+          volumeMounts:
+            - name: resolute-server-update-schema
+              mountPath: /tmp/update_schema.sql
+              subPath: update_schema.sql
+          env:
+            - name: userPassword
+              value: heartland-ocean-praetor-neat
```

Comment:
If it's a confidential value, should we put it in a kubernetes secret, like the following? Or no need because testnet only>

```
    env:
      - name: PGPASSWORD
        valueFrom:
          secretKeyRef:
            name: postgresql-secret
            key: password
  ```

Reply:
it should actually but the secret already exposed in other place in the same manifest
only mainnet one is using secret manager now for db
we can mark it as to-do

Reply:
Ok! Sure no problem!